docs/rules/computed-property-return.md

```diff
@@ -0,0 +1,44 @@
+# Enforces that a return statement is present in computed property (computed-property-return)
+
+## :book: Rule Details
+
+This rule enforces that a `return` statement is present in `computed` properties.
+
+:-1: Examples of **incorrect** code for this rule:
+
+```js
+export default {
+  computed: {
+    foo () {
+    },
+    bar: function () {
+    }
+  }
+}
+```
+
+:+1: Examples of **correct** code for this rule:
+
+```js
+export default {
+  computed: {
+    foo () {
+      return true
+    },
+    bar: function () {
+      return false
+    }
+  }
+}
+```
+
+## :wrench: Options
+
+This rule has an object option:
+- `"treatUndefinedAsUnspecified"`: `true` (default) allows implicitly returning undefined with a `return;` statement.
```

Comment:
damn description is invalid, i forgot to change it, by default its treating undefined as unspecified than `return;` will thrown an error, but i prefer to leave a choice to disable it because undefined can be useful in some cases.